include/pm/pm.h

```diff
@@ -78,16 +78,13 @@ struct pm_notifier {
  */
 void pm_power_state_force(struct pm_state_info info);
 
-#ifdef CONFIG_PM_DEBUG
 /**
  * @brief Dump Low Power states related debug info
  *
  * Dump Low Power states debug info like LPS entry count and residencies.
  */
 void pm_dump_debug_info(void);
```

Comment:
Since this is a public function, the dummy implementation should be provided as a static inline in the header, as we do for similar cases, that is, remove https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/pm/power.c#L71 and do something like https://github.com/zephyrproject-rtos/zephyr/blob/main/include/pm/device.h#L212

Reply:
Thank you for your comments, updated.